TestFoundation/TestNSString.swift

```diff
@@ -1169,6 +1170,17 @@ class TestNSString : XCTestCase {
         let s6 = NSString(string: "Beyonce\u{301} and Tay")
         XCTAssertEqual(s6.substring(with: NSMakeRange(7, 9)), "\u{301} and Tay")
     }
+    
+    func test_createCopy() {
+        //SR-6139
```

Comment:
This comment isn't particularly relevant - as long as the SR-6139 is mentioned in the commit message, people will be able to identify that it came from this bug in the first place. Can you remove this comment?